packages/wonder-blocks-popover/components/popover-dialog.test.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from "react";
+
+import {TooltipTail} from "@khanacademy/wonder-blocks-tooltip";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import PopoverDialog from "./popover-dialog.js";
+import PopoverContentCore from "./popover-content-core.js";
+
+jest.mock("@khanacademy/wonder-blocks-tooltip");
+
+describe("PopoverDialog", () => {
+    afterEach(() => {
+        unmountAll();
```

Comment:
@kevinbarabash What do you think about adding your cool hack so that this just happens and our use of our custom mount/unmount is hidden away to reduce boilerplate?

No action for this diff.

Reply:
I think we should.

Reply:
Interesting! How could we make that work?

Reply:
In jest.config.js in webapp we do:
```
    moduleNameMapper: {
        "^enzyme$": "<rootDir>/dev/tools/jest-runner/enzyme-shim.js",
        // Load a .js file with no exports whenever a .css or .less file is requested.
        "\\.(css|less)$": "<rootDir>/dev/tools/jest-runner/style-mock.js",
    },
```
The shim there is very similar to shim here, see https://github.com/Khan/webapp/blob/master/dev/tools/jest-runner/enzyme-shim.js for details.